myapp/app/models/task.rb

```diff
@@ -1,8 +1,10 @@
 class Task < ApplicationRecord
-  validates :name, presence: true, length: { maximum: 255 }
+  validates :name, presence: true, length: { maximum: 50 }
```

Comment:
I think it was too long and possible to make layout off.
Then I cap the length.